src/test/java/AlertsTest.java

```diff
@@ -0,0 +1,37 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.pompages.AlertsPomPage;
+
+public class AlertsTest extends BaseTest {
+    @Test
+    public void jsAlert() {
+        AlertsPomPage alertsPomPage = new AlertsPomPage(driver);
+        alertsPomPage.openAlertsPage();
+        alertsPomPage.pressJSAlert();
+        String actualText = driver.switchTo().alert().getText();
```

Comment:
Let's move 'String actualText = driver.switchTo().alert().getText();' to alertsPage